shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -20,7 +20,8 @@
 
 FlutterPlatformViewLayer::~FlutterPlatformViewLayer() = default;
 
-FlutterPlatformViewsController::FlutterPlatformViewsController() = default;
+FlutterPlatformViewsController::FlutterPlatformViewsController()
+    : layer_pool_(std::make_unique<FlutterPlatformViewLayerPool>()){};
```

Comment:
Empty constructor makes a nullptr too.

Reply:
discussed offline. Changing that resulted in a bad memory access

Reply:
To be clear omitting the initializer resulted in an error, an empty initializer didn't.